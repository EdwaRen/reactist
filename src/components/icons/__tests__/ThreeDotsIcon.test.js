import React from 'react'
import { shallow } from 'enzyme'

import ThreeDotsIcon from '../ThreeDotsIcon.svg'

describe('ThreeDotsIcon', () => {
    it('renders a svg icon', () => {
        const threeDotsIcon = shallow(<ThreeDotsIcon />)
        expect(threeDotsIcon).toMatchSnapshot()
    })
})
