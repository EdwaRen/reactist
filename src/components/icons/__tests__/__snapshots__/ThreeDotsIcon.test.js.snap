// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreeDotsIcon renders a svg icon 1`] = `
<svg
  height="26px"
  viewBox="0 0 26 26"
  width="26px"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <circle
      cx="19"
      cy="13"
      fill="#A5A5A5"
      r="2"
    />
    <circle
      cx="13"
      cy="13"
      fill="#A5A5A5"
      r="2"
    />
    <circle
      cx="7"
      cy="13"
      fill="#A5A5A5"
      r="2"
    />
  </g>
</svg>
`;
